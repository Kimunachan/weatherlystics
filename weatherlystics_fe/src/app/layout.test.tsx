import {
  render,
  screen,
  waitForElementToBeRemoved,
} from "@/utils/customTestUtils";
import RootLayout from "./layout";

describe("Root Layout", () => {
  it("should render the root layout", async () => {
    const { container } = render(
      <RootLayout children={<div data-testid="dummy"></div>} />
    );
    expect(container).toMatchSnapshot();
    expect(screen.getByTestId("dummy")).toBeInTheDocument();
  });
});
