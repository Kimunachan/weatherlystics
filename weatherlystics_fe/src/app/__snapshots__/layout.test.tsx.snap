// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root Layout should render the root layout 1`] = `
<div>
  <html
    lang="de"
  >
    <body>
      <div
        class="Toastify"
      />
      <div
        data-testid="dummy"
      />
    </body>
  </html>
</div>
`;
